svn/common_base.py

```diff
@@ -30,6 +33,13 @@ def external_command(
         else:
             kwargs['stderr'] = subprocess.STDOUT
 
+        # This allows correct decoding of commit messages on Windows
+        # subprocess.Popen defaults to locale.getpreferredencoding()
+        # which can be cp1252 (for example)
+        if (decode_text and platform.system() == "Windows"
+                and sys.version_info >= (3, 6)):
```

Comment:
This version seems arbitrary. It's not good practice to hard-code specific versions. If absolutely necessary, it's better to test for the features/characteristics that you actually need. In this case, you can just pass `universal_newlines=True`, which is supported in both 2.7 and 3.x .

Reply:
Since we're already passing `universal_newlines` (https://github.com/dsoprea/PySvn/blob/master/svn/common_base.py#L39), it will already always be returned as UTF-8 on Python 2.7 and `string` in Python 3.x in situations where text is expected. This pull-request is either in response to a not-understand problem or it's a symptom of a different problem. In which context are you encountering this?

Reply:
Please open an issue with your usage and the error that you're encountering and go from there.